src/openrct2/CmdlineSprite.cpp

```diff
@@ -575,18 +629,41 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
                 return -1;
             }
             std::string strPath = Json::GetString(path);
+            GamePalette spritePalette = DefaultPalette;
+            ImageImporter::ImportFlags flags = ImageImporter::ImportFlags::RLE;
 
+            // {"x": n, "y", n} in parkobj
             json_t x_offset = jsonSprite["x_offset"];
             json_t y_offset = jsonSprite["y_offset"];
 
-            auto palette = (Json::GetString(jsonSprite["palette"]) == "keep") ? ImageImporter::Palette::KeepIndices
-                                                                              : ImageImporter::Palette::OpenRCT2;
-            bool forceBmp = !jsonSprite["palette"].is_null() && Json::GetBoolean(jsonSprite["forceBmp"]);
+            // {"format": "raw"} in parkobj
+            if (Json::GetBoolean(jsonSprite["forceBmp"]))
+            {
+                flags = ImageImporter::ImportFlags::BMP;
+            }
+
+            if (jsonSprite["palette"].is_string())
+            {
+                std::string paletteString = Json::GetString(jsonSprite["palette"]);
+                if (paletteString == "keep")
+                {
+                    flags = static_cast<ImageImporter::ImportFlags>(flags | ImageImporter::ImportFlags::KeepIndices);
+                }
+                else if (paletteString == "green" || paletteString == "openrct2")
```

Comment:
you use a case insensitive compare in `GetPaletteFromString` but a case sensitive compare here. Stick to one. (probably just the case sensitive one)